src/app/test-record/test-record.component.ts

```diff
@@ -17,6 +17,7 @@ import { Preparer } from '@app/models/preparer';
 import { DialogBoxComponent } from '@app/shared/dialog-box/dialog-box.component';
 import { TestStation } from '@app/models/test-station';
 import { PreventLeavePageModalComponent } from '@app/shared/prevent-page-leave-modal/prevent-leave-page-modal.component';
+import { TestDeleteReasonModalComponent } from './test-delete-reason-modal/test-delete-reason-modal.component';
```

Comment:
Would TestDeleteModal naming sound better? I think it's shorter and it's not that important to mention the reason bit

Reply:
i followed adr naming one. i think we will remove them in the future anyway so i don't know if it is needed for change. let me know